### DISCUSSION TITLE
Need an Invoice

### DISCUSSION THREAD

**Discussion by Josianne-Lafond** [aclap-dev/video-downloadhelper#1218] [License & Purchase requests]
### Your question

I bought last July a premium version and I didn't receive any invoice for this purchase. Can you please send me this invoice or tell me how I can get it. Purchase was done July 25th.
Thank you

### Addon & CoApp version and details

version: 9.1.0.42
target: mozilla
channel: stable
lang: fr
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"fd064f13f4bf4430"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:129.0) Gecko/20100101 Firefox/129.0
{
    "valid_license_message_has_been_displayed": true,
    "download_directory": "C:\\Users\\jlcon\\dwhelper",
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": true,
    "open_count_store": 1,
    "successfull_dl": 4
}

**Comment by jcfrog** [comment]
I sent you an email.

**Reply by samotter** [reply]
Hello! I too need an invoice. I purchased premium last week but can't find an email anywhere. How can I contact you?

**Reply by jcfrog** [reply]
Hi, send us an email at licenses@downloadhelper.net
